tests/dummy/app/styles/app.scss

```diff
@@ -34,7 +43,7 @@ html {
 
 body {
   height: 200vh;
-  background-color: $c-body-bg;
+  background-color: _e3dn('body-bg');
```

Comment:
You're right, we can add a `body` tag in the addon styles. However, now that I'm seeing this, I think it may be a little far reaching and the setting should be removed. Ideally, this nav should be attached to someone's site. They shouldn't have to change their body's background for the nav to work. Example: They may have a white background, but still want say a green header and a blue nav.

I think we should remove `'body-bg'` from config. Let's make the config only the trigger and nav, since those are the components the addon adds.